contracts/SimpleOperations.sol

```diff
@@ -170,39 +186,16 @@ contract SimpleOperations is Operations {
 	// Getters
 
 	function isLatest(bytes32 _client, bytes32 _release)
-		public
+		external
 		view
 		returns (bool)
 	{
-		return clientExists(_client) && latestInTrack(_client, track(_client, _release)) == _release;
-	}
-
-	function track(bytes32 _client, bytes32 _release)
-		public
-		view
-		returns (uint8)
-	{
-		if (clientExists(_client)) {
-			return client[_client].release[_release].track;
-		} else {
-			return 0;
-		}
-	}
-
-	function latestInTrack(bytes32 _client, uint8 _track)
-		public
-		view
-		returns (bytes32)
-	{
-		if (clientExists(_client)) {
-			return client[_client].current[_track];
-		} else {
-			return 0;
-		}
+		return clientExists(_client) &&
+			latestInTrack(_client, track(_client, _release)) == _release;
```

Comment:
`latestInTrack` is external, are you sure we can call it from here?

Reply:
It's defined as `external` in the interface (it's mandatory) but it's overridden here as `public`. (Also these changes come from #10).